### DISCUSSION TITLE
Info / alert panels in-page

### DISCUSSION THREAD

**Discussion by nicedisguise** [twilio-labs/paste#1563] [General] [answered]
Hey folks,

Both myself and Porlin are working on a similar layout in the Flex area of console and need a similar component. Basically an in-page info/alert box. 

My example - using card style: https://www.figma.com/file/0XCJt2dfb7026AMMm6va7v/email-integration-in-flex-and-conversations-mapping?node-id=3692%3A753
Porlin's example - using alert style: https://www.figma.com/file/Zq33a5EhHqxj5gFLLDISz0/Flex-Console?node-id=18%3A22766

We were basically wondering what's the thoughts around this specific area? As an aside it's a convention that's also used in Flex so It'd be great to use a consistant pattern.

Cheers

Duncan

**Comment by richbachman** [comment]
Hi @nicedisguise I'll share this with the team and gather some feedback.

**Comment by richbachman** [comment, marked as answer]
Hi @nicedisguise, the team chatted about this and we feel you have a good use-case for some sort of callout component. Basically this would be like an alert but wouldn't announce the content to screen readers. It would just contain ever-present useful information that a user should know about in order to complete a specific task.

Since that component doesn't exist yet, you could use an Alert for now, and swap it out later. That could also be composed using a Card, icon, and text. If you go that route, schedule an office hours visit or post here again so the team can review.

Have either of these ideas been user tested to see which one a user prefers? If so, we'd like to hear more about that.

**Reply by nicedisguise** [reply]
Thanks for the answer :) 
My preference is maybe to lean on the alert style as it breaks the page up a bit. I'll refine this a bit and send you the developments - as I said this is a pattern we use on Flex so I'll be happy to work with you to develop this one.

Regarding testing: Not yet - will be tested with users soon hopefully.

**Reply by richbachman** [reply]
Sound great. Thanks!
